test/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore.Test/EntityFrameworkServicesTests.cs

```diff
@@ -70,54 +70,61 @@ private EntityFrameworkServices GetEfServices(string path, string applicationNam
 
         }
 
-        private IProjectContext GetProjectContext(string path, bool isMsBuild)
+        private ProjectInformation GetProjectInformation(string path, bool isMsBuild)
         {
             if (isMsBuild)
             {
-                return new MsBuildProjectContextBuilder()
+                // TODO needs to be fixed to get all dependent Projects as well.
+                return new ProjectInformation(new MsBuildProjectContextBuilder()
```

Comment:
Optional nit: code style a little hard to read. Suggest making a local variable.